docs/maps/search.asciidoc

```diff
@@ -35,13 +36,29 @@ image::maps/images/global_search_bar.png[]
 [[maps-create-filter-from-map]]
 === Create filters from a map
 
-You can create two types of filters by interacting with your map:
+You can create three types of filters by interacting with your map:
 
-* <<maps-spatial-filters, Spatial filters>>
-* <<maps-phrase-filter, Phrase filters>>
+* <<maps-map-extent-filter>>
+* <<maps-spatial-filters>>
+* <<maps-phrase-filter>>
+
+[float]
+[[maps-map-extent-filter]]
+==== Filter dashboard by map extent
+
+Filter dashboard panels by map extent to show uniform data across all panels.
```

Comment:
"Filter dashboard panels by map extent" sounds like an implementation detail.  Do you need to say that here or is enough to say:

A map extent filter* shows uniform data across all panels.  As you pan and zoom your map, all panels will update to only include data that is visible in your map.

*Or another appropriate name for the filter.